build/helper.py

```diff
@@ -38,13 +38,24 @@ def extract_input_parameters(parameters, sessionName = 'vi'):
     return [x for x in parameters if x['direction'] == 'in' and x['name'] != sessionName]
 
 def extract_output_parameters(parameters):
```

Comment:
How does it no longer match? Because it is output-params minus ivi-dance-params?